mobile/src/main/java/org/openhab/habdroid/model/Widget.java

```diff
@@ -140,9 +143,14 @@ public Widget build() {
             maxValue(Math.max(minValue(), maxValue()));
             step(Math.abs(step()));
 
+            if (state() == null && item() != null) {
+                state(item().state());
+            }
```

Comment:
Can remove this, handled by determineWidgetState already. (Need to pass item state into state() for OH1)